awx_collection/tools/roles/template_galaxy/templates/README.md.j2

```diff
@@ -109,7 +109,8 @@ The following notes are changes that may require changes to playbooks:
  - The `notification_configuration` parameter of `tower_notification_template` has changed from a string to a dict. Please use the `lookup` plugin to read an existing file into a dict.
  - `tower_credential` no longer supports passing a file name to `ssh_key_data`.
  - The HipChat `notification_type` has been removed and can no longer be created using the `tower_notification_template` module.
- - Lookup plugins now always return a list, and if you want a scalar value use `lookup` as opposed to `query`
+ - Lookup plugins now always return a list, and if you want a scalar value use `lookup` as opposed to `query`.
+ - Removed `oauth_token` as an option for authentication. Internal logic no longer creates a token if given username and password.
```

Comment:
I added this already on line 72, not sure how we want to handle this

Reply:
Ok yeah, I didn't see that. Oh, and on line 79 as well.